doc/user/content/sql/system-catalog/mz_internal.md

```diff
@@ -170,25 +170,6 @@ any kind of capacity planning.
 | `disk_bytes`           | [`uint8`]   | The disk allocation per process, if the replica has a [disk](/sql/create-cluster#disk) attached. `NULL` otherwise. |
 | `credits_per_hour`     | [`numeric`] | The number of compute credits consumed per hour.                                                                                                             |
 
-### `mz_cluster_links`
```

Comment:
Are we removing the table with the deprecation of the option, or only when we remove it? We'll need to remove the SQL Logic Tests if we remove this, too.

Reply:
Not sure yet, but I figured we might as well remove all references to linked clusters while we do the deprecation and eventual removal. I adjusted the docs linter here to expect `mz_cluster_links` to be undocumented.

Reply:
That's 👌.